AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+||assets.f-secure.com/fs-cookie-consent.js
```

Comment:
This site is on the https exclusion list. This won't work for apps.
However, we can replace the old rule.